pkg/registry/servicecatalog/clusterserviceclass/storage.go

```diff
@@ -144,8 +146,32 @@ func NewStorage(opts server.Options) (rest.Storage, rest.Storage) {
 		CreateStrategy: clusterServiceClassRESTStrategies,
 		UpdateStrategy: clusterServiceClassRESTStrategies,
 		DeleteStrategy: clusterServiceClassRESTStrategies,
-		Storage:        storageInterface,
-		DestroyFunc:    dFunc,
+
+		TableConvertor: tableconvertor.NewTableConvertor(
+			[]metav1beta1.TableColumnDefinition{
+				{Name: "Name", Type: "string", Format: "name"},
+				{Name: "External name", Type: "string"},
+				{Name: "Broker", Type: "string"},
+				{Name: "Bindable", Type: "bool"},
+				{Name: "Plan updatable", Type: "bool"},
```

Comment:
Hrm - reading these, I'm not certain what the naming convention is for kubectl columns.

Reply:
They use two different forms: 
- `Cluster-IP`, `OS-Image`, `Kernel-Version`, `Container-Runtime`, ...
- `Node Selector`, `Last Schedule`, `Access Modes`,`Reclaim Policy`

See: https://github.com/kubernetes/kubernetes/blob/7eb88f11d23d2be7dc3a91f727a1a77a0abac5e8/pkg/printers/internalversion/printers.go#L76-L422